src/components/svgs/TLogoCircle.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+const TLogoCircle = ({ size = 104, fill = "#080503" }) => {
```

Comment:
Probably prefer for this to be `TbtcLogo` or similar, given that's still the right term.